classes/classes/Items/Mutations.as

```diff
@@ -1471,7 +1471,7 @@
 				player.vaginas[0].vaginalLooseness = VAGINA_LOOSENESS_TIGHT;
 				player.vaginas[0].vaginalWetness = VAGINA_WETNESS_NORMAL;
 				player.vaginas[0].virgin = true;
-				player.clitLength = .25;
+				player.getClitLength(.25)
```

Comment:
Thanks for spotting that.
It unfortunately compiles because actionscript performs implicit casts for function arguments (without a warning no less).
Will go over the entire 	00ce9cc commit to see if there are more errors.